src/lspclient.rs

```diff
@@ -0,0 +1,12 @@
+use bip32::secp256k1::PublicKey;
```

Comment:
might make sense to have this as a separate library

Reply:
damn nice catch that is not supposed to be in there. Code imports messed up. Tired of every single library redefining their own PublicKey.

Reply:
oh lol, didn't mean to catch that :joy: 

meant that the lsp http client could be a separate library

Reply:
Oh yeah totally. I thought about that as I created the file. I'm unsure of how it'll end up as time goes on. Could have a voltage specific one, which is what it'll be originally, but what if we want to change it to be our own type or more along the lines of the LSP spec. Would rather wait to see how it turns out or until someone asks.